### PR TITLE
Lingohub sync and additional fixes

### PR DESCRIPTION
**Pull request by sampaiodiego**
This is a manual lingohub sync since lingohub fetches from github are disabled for languages other than english. after merging this I'll enable it back again.

I've made yet another fixes on the spacing of placeholders using [this script](https://gist.github.com/sampaiodiego/c72b0018dcd5f309b7b96442b484576c).

All _new_ lingohub translations are at the bottom of the i18n files. To fix that I'll import the translations to lingohub (after merging) and then grab them **from** lingohub back to github, this will fix the order of the keys.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/id.i18n.json [hunk 1]
```diff
@@ -1315,7 +1315,7 @@
   "Importer_CSV_Information": "Importir CSV memerlukan format tertentu, baca dokumentasi bagaimana menyusun file zip Anda:",
   "Importer_done": "Mengimpor lengkap!",
   "Importer_finishing": "Menyelesaikan impor.",
-  "Importer_From_Description": "Impor __from__ Data 's menjadi Rocket.Chat.",
```
Comment (engelgabriel) [inline packages/rocketchat-i18n/i18n/id.i18n.json]: fix

File: packages/rocketchat-i18n/i18n/km.i18n.json [hunk 2]
```diff
@@ -1430,7 +1430,7 @@
   "Importer_CSV_Information": "អ្នកនាំចូល CSV ត្រូវការទ្រង់ទ្រាយជាក់លាក់សូមអានឯកសារសម្រាប់របៀបរចនាសម្ព័ន្ធឯកសារ zip របស់អ្នក:",
   "Importer_done": "ការនាំចូលបានបញ្ចប់!",
   "Importer_finishing": "បញ្ចប់ការនាំចូល។",
-  "Importer_From_Description": "ការនាំចូល __from__ ទិន្នន័យ's បានចូលទៅក្នុង Rocket.Chat ។",
```
Comment (engelgabriel) [inline packages/rocketchat-i18n/i18n/km.i18n.json]: fix

File: packages/rocketchat-i18n/i18n/ja.i18n.json [hunk 3]
```diff
@@ -3119,5 +3119,62 @@
...
+  "thread-welcome": "スレッドを作成してくれてありがとう、__username__！ __parentChannel__に何人かのメンバーを招待しました。 ヒント：長い間何も起こらない場合は、\"@all\" で彼らに気がつかせることができます。",
+  "thread": "スレッド",
+  "Threading_context_menu_button": "専用ボタン",
+  "Threading_context_menu_none": "非表示",
+  "Threading_description": "何が起こっているのかについての概要を把握するのを手伝ってください！ スレッドを作成することで、選択したもののサブチャンネルが作成され、両方がリンクされます。",
+  "Threading_first_message_title": "あなたのメッセージ",
+  "Threads_in_sidebar": "サイドバーにスレッドのカテゴリを表示する",
+  "Threads": "スレッド",
+  "Threading_title": "新しいスレッドを作る",
+  "To_install_the_new_version_of_RocketChat_Livechat_in_your_website_copy_paste_this_code_above_the_last_body_tag_on_your_site": "Webサイトへ<strong>新しいバージョン</strong>のRocket.Chat ライブチャットを導入するには、あなたのサイトの最後の <strong>&lt;/body&gt;</strong> タグの前に、このコードをコピー &amp; ペーストしてください。",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/ja.i18n.json]: `Webサイトへ<strong>新しいバージョン</strong>のRocket.Chat` missing space after </strong> here

File: packages/rocketchat-i18n/i18n/ko.i18n.json [hunk 4]
```diff
@@ -253,7 +253,7 @@
   "Analytics_Google": "구글 애널리틱스",
   "Analytics_Google_id": "Tracking ID",
   "and": "그리고",
-  "And_more": "그리고 __length __만큼 더",
+  "And_more": "그리고 __length__만큼 더",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/ko.i18n.json]: Missing a space here after `__length__` maybe? :thinking:

File: packages/rocketchat-i18n/i18n/ko.i18n.json [hunk 5]
```diff
@@ -1384,7 +1384,7 @@
   "I_saved_my_password_close_this_message": "비밀번호를 저장했습니다. 이 메시지를 닫습니다.",
   "Idle_Time_Limit": "유휴 시간 제한",
   "Idle_Time_Limit_Description": "상태가 사라질 때까지의 시간. 값은 초 단위 여야합니다.",
-  "if_they_are_from": "(그들이  %s 출신 인 경우)",
+  "if_they_are_from": "(그들이 % s 출신 인 경우)",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/ko.i18n.json]: Unnecessary space  here

File: packages/rocketchat-i18n/i18n/ku.i18n.json [hunk 6]
```diff
@@ -1378,12 +1378,12 @@
...
+  "Integration_Run_When_Message_Is_Edited_Description": "Piştre ku peyama veguhastin divê yek integration rabe? Sazkirina vê çewtiyê dê bibe sedema întegrasyonê ku tenê bi <strong>nû</strong> peyamên rêvebirin.",
   "Integration_updated": "Integration cara ghurrînikarî bi serda",
   "Integration_Word_Trigger_Placement": "Her derheqê peyivînê",
   "Integration_Word_Trigger_Placement_Description": "Ma Peyvê ku di destpêka destpêkê de ji hev re veşartî bikişîne?",
   "Integrations": "تێهەڵکێشان",
-  "Integrations_for_all_channels": "Enter <strong>all_public_channels</strong>Ji bo hemû kanalên gelemperî guhdar bikin, <strong>all_private_groups</strong>ji bo guhdariya hemû taybet û guhdariya hemû Lîsteyên tevahî peyamên <strong>guhdar bikin.",
+  "Integrations_for_all_channels": "Enter <strong>all_public_channels</strong>Ji bo hemû kanalên gelemperî guhdar bikin, <strong>all_private_groups</strong> ji bo guhdariya hemû taybet û guhdariya hemû Lîsteyên tevahî peyamên <strong>guhdar bikin.",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/ku.i18n.json]: Missing space after first `</strong>` and closing the last `<strong>` tag.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 7]
```diff
@@ -2029,10 +2029,10 @@
   "Prune_finished": "Пруне је завршио",
   "Prune_Messages": "Пруне Поруке",
   "Prune_Modal": "Да ли сте сигурни да желите да обришете ове поруке? Обрезане поруке се не могу опоравити.",
-  "Prune_Warning_all": "Ово ће избрисати све %с у %с!",
-  "Prune_Warning_before": "Ово ће избрисати све %с у %с прије %с.",
-  "Prune_Warning_after": "Ово ће избрисати све %с за %с након %с.",
-  "Prune_Warning_between": "Ово ће избрисати све %с у %с између %с и %с.",
+  "Prune_Warning_all": "Ово ће избрисати све %s у %s!",
+  "Prune_Warning_before": "Ово ће избрисати све %s у %s прије %s.",
+  "Prune_Warning_after": "Ово ће избрисати све %s за %s након% с.",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: `%c` forgot here.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 8]
```diff
@@ -2205,7 +2205,7 @@
   "run-migration_description": "Дозвола за покретање миграција",
   "Running_Instances": "Покретање инстанци",
   "Runtime_Environment": "Рунтиме Енвиронмент",
-  "S_new_messages_since_s": "% с нових порука од% с",
+  "S_new_messages_since_s": "% с нових порука од %s",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: `%c` forgot here.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 9]
```diff
@@ -2333,15 +2333,15 @@
   "Size": "величина",
   "Skip": "прескочити",
   "Slack_Users": "Слацкови корисници ЦСВ",
-  "SlackBridge_error": "СлацкБридге је добио грешку при увозу ваших порука на% с:% с",
-  "SlackBridge_finish": "СлацкБридге је завршио са увозом порука на% с. Поново учитај за приказ свих порука.",
+  "SlackBridge_error": "СлацкБридге је добио грешку при увозу ваших порука на %s:% с",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: `%c` forgot here.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 10]
```diff
@@ -2333,15 +2333,15 @@
...
+  "SlackBridge_finish": "СлацкБридге је завршио са увозом порука на %s. Поново учитај за приказ свих порука.",
   "SlackBridge_Out_All": "СлацкБридге Оут Алл",
   "SlackBridge_Out_All_Description": "Слање порука из свих канала који постоје у Слацк-у и бот се придружио",
   "SlackBridge_Out_Channels": "СлацкБридге Оут Цханнелс",
   "SlackBridge_Out_Channels_Description": "Изаберите који канали ће послати поруке назад у Слацк",
   "SlackBridge_Out_Enabled": "СлацкБридге Оут је омогућен",
   "SlackBridge_Out_Enabled_Description": "Изаберите да ли СлацкБридге такође треба да пошаље ваше поруке на Слацк",
-  "SlackBridge_start": "@% с је покренуо СлацкБридге увоз на `#% с`. Обавестићемо вас када заврши.",
+  "SlackBridge_start": "@ %s је покренуо СлацкБридге увоз на `#% с`. Обавестићемо вас када заврши.",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: `%c` forgot here.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 11]
```diff
@@ -2449,13 +2449,13 @@
...
   "The_channel_name_is_required": "Име канала је потребно",
   "The_emails_are_being_sent": "Е-маил поруке се шаљу.",
-  "The_field_is_required": "је обавезно поље% с.",
+  "The_field_is_required": "је обавезно поље %s.",
   "The_image_resize_will_not_work_because_we_can_not_detect_ImageMagick_or_GraphicsMagick_installed_in_your_server": "Ресизе слика неће радити, јер не можемо открити ИмагеМагицк или грапхицсмагицк инсталиран на вашем серверу.",
   "The_redirectUri_is_required": "РедирецтУри је потребна",
-  "The_server_will_restart_in_s_seconds": "Сервер ће се поново покренути у% с секунди",
-  "The_setting_s_is_configured_to_s_and_you_are_accessing_from_s": "<strong>Поставка% с</strong> конфигурисан <strong>да% с</strong> и приступате <strong>из% с!</strong>",
+  "The_server_will_restart_in_s_seconds": "Сервер ће се поново покренути у %s секунди",
+  "The_setting_s_is_configured_to_s_and_you_are_accessing_from_s": "<strong>Поставка %s</strong> конфигурисан <strong>да %s</strong> и приступате <strong>из% с!</strong>",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: `%c` forgot here too.

File: packages/rocketchat-i18n/i18n/ta-IN.i18n.json [hunk 12]
```diff
@@ -816,7 +816,7 @@
   "Create_unique_rules_for_this_channel": "இந்த சேனலுக்கான தனிப்பட்ட விதிகள் உருவாக்கவும்",
   "Created_at": "மணிக்கு உருவாக்கப்பட்டது",
   "Created_at_s_by_s": "<strong>% கள்</strong> <strong>மூலம் %s</strong> இல் உருவாக்கப்பட்டது",
-  "Created_at_s_by_s_triggered_by_s": "<strong>%s</strong><strong>%s </strong><strong>%s</strong>தூண்டுதலாக %sஇல் உருவாக்கப்பட்டது",
+  "Created_at_s_by_s_triggered_by_s": "<strong>%s</strong><strong>%s</strong><strong>%s</strong> தூண்டுதலாக %sஇல் உருவாக்கப்பட்டது",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/ta-IN.i18n.json]: Maybe missing space after the first and second `</strong>`? :thinking:

File: packages/rocketchat-i18n/i18n/tr.i18n.json [hunk 13]
```diff
@@ -2223,9 +2223,9 @@
   "Prune_Messages": "Kuru Yazılı Mesajlar",
   "Prune_Modal": "Bu mesajları ertelemek istediğinden emin misin? Budanmış mesajlar kurtarılamaz.",
   "Prune_Warning_all": "Bu %s içindeki tüm %s leri silecek!",
-  "Prune_Warning_before": "Bu %s içindeki tüm %s leri %s'den önce siler.",
-  "Prune_Warning_after": "Bu %s içindeki tüm %s leri %s sonra siler.",
-  "Prune_Warning_between": "Bu, %s içindeki tüm %s leri %s ve %s arasında siler.",
+  "Prune_Warning_before": "Bu %s içindeki tüm %sleri %s'den önce siler.",
+  "Prune_Warning_after": "Bu %s içindeki tüm %sleri %s sonra siler.",
+  "Prune_Warning_between": "Bu, %s içindeki tüm %sleri %s ve %s arasında siler.",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/tr.i18n.json]: Missing space after first `%s`.

File: packages/rocketchat-i18n/i18n/tr.i18n.json [hunk 14]
```diff
@@ -3095,5 +3095,68 @@
...
+  "New_thread_name": "Konu odası için anlamlı bir ad",
+  "No_replies_yet": "Henüz bir yanıt yok",
+  "No_threads_yet": "Henüz bir konu yok",
+  "Parent_channel_doesnt_exist": "Channel mevcut değil.",
+  "Phone_number": "Telefon ",
+  "Please_fill_an_email": "Lütfen bir e-posta girin",
+  "Replies": "Yanıtlar",
+  "RetentionPolicy_DoNotExcludeThreads": "Konu iletilerini dışarıda tutmayın",
+  "Show_on_offline_page": "Çevr",
+  "Sync / Import": "Eşitle / İçe Aktar",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/tr.i18n.json]: Is this a valid key? :thinking:

File: packages/rocketchat-i18n/i18n/vi-VN.i18n.json [hunk 15]
```diff
@@ -2650,15 +2650,15 @@
...
+  "User_sent_a_message_on_channel": "<strong>__người dùng__</strong> đã gửi một tin nhắn về <strong>__channel__</strong>:",
+  "User_sent_a_message_to_you": "<strong>__người dùng__</strong> đã gửi cho bạn một tin nhắn:",
   "user_sent_an_attachment": "__user__ đã gửi tập tin đính kèm",
   "User_Settings": "Thiết lập người dùng",
   "User_unmuted_by": "Người dùng <em>__user_unmuted__</em>đã được bật bởi <em>__user_by__</em>.",
   "User_unmuted_in_room": "User unmuted trong phòng",
   "User_updated_successfully": "Người dùng đã cập nhật thành công",
-  "User_uploaded_a_file_on_channel": "<strong>__người dùng__</strong>đã tải lên một tệp tin trên <strong>__channel__</strong>:",
-  "User_uploaded_a_file_to_you": "<strong>__username__</strong>đã gửi cho bạn tệp:",
+  "User_uploaded_a_file_on_channel": "<strong>__người dùng__</strong> đã tải lên một tệp tin trên <strong>__channel__</strong>:",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/vi-VN.i18n.json]: Keyword `__username__`  has been translated here.

File: packages/rocketchat-i18n/i18n/zh-TW.i18n.json [hunk 16]
```diff
@@ -250,7 +250,7 @@
   "Analytics_Google": "谷歌分析",
   "Analytics_Google_id": "跟踪號碼",
   "and": "和",
-  "And_more": "而更多的__length __",
+  "And_more": "而更多的__length__",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/zh-TW.i18n.json]: Missing space here before `__length__`.

File: packages/rocketchat-i18n/i18n/zh-TW.i18n.json [hunk 17]
```diff
@@ -1319,7 +1319,7 @@
   "Importer_CSV_Information": "CSV導入程序需要特定的格式，請閱讀文檔以了解如何構建您的zip文件：",
   "Importer_done": "導入完成！",
   "Importer_finishing": "完成了進口。",
-  "Importer_From_Description": "進口__from __的數據轉換成Rocket.Chat。",
+  "Importer_From_Description": "進口__from__的數據轉換成Rocket.Chat。",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/zh-TW.i18n.json]: Missing spaces here.

File: packages/rocketchat-i18n/i18n/sr.i18n.json [hunk 18]
```diff
@@ -554,7 +554,7 @@
   "Conversation": "Разговор",
   "Conversation_closed": "Разговор затворена: __цоммент__.",
```
Comment (MarcosSpessatto) [inline packages/rocketchat-i18n/i18n/sr.i18n.json]: I think the keyword has been translated here.

File: packages/rocketchat-i18n/i18n/tr.i18n.json (hunk 14 above)
Comment (sampaiodiego) [reply packages/rocketchat-i18n/i18n/tr.i18n.json]: seems to be =)

**Pull request merged**